src/test/shell/bazel/BUILD

```diff
@@ -73,6 +75,20 @@ filegroup(
     ],
 )
 
+genrule(
+    name = "junit_jar",
+    srcs = ["@maven//:junit_junit_file"],
```

Comment:
Doesn't maven_import make the jar available directly as a label?

Reply:
The problem is that we need to find those jars during shell test setup using rlocation: https://cs.opensource.google/bazel/bazel/+/master:src/test/shell/testenv.sh.tmpl;l=118-124

Ideally, I need to be able to reference those jars using a stable runfile path (without version number).